packages/block-editor/src/components/text-alignment-control/README.md

```diff
@@ -0,0 +1,54 @@
+# TextAlignmentControl
+
+The `TextAlignmentControl` component is responsible for rendering a control element that allows users to select and apply text align options to blocks or elements in the Gutenberg editor. It provides an intuitive interface for changing the text alignment by applying different alignments such as `left`, `center`, `right`.
+
+## Development guidelines
+
+### Usage
+
+Renders the Text Align Component with `left`, `center`, `right` options.
+
+```jsx
+import { TextAlignmentControl } from '@wordpress/block-editor';
+
+const MyTextAlignmentControlComponent = () => (
+	<TextAlignmentControl
+		value={ textTransform }
+		onChange={ ( value ) => {
+			setAttributes( { textTransform: value } );
+		} }
+	/>
+);
+```
+
+### Props
+
+The component accepts the following props.
+
+#### value
+
+The current value of the Text Alignment setting. Available options are `left|center|right`.
+
+-   Type: `String`
+-   Required: No
+
+#### controls
+
+An array of strings for what controls to show, by default it shows all. Available options are `left|center|right`.
```

Comment:
Is `justify` a valid option?

Reply:
> Consumers can also opt-in to justify by overriding the controls prop.

This [comment](https://github.com/WordPress/gutenberg/pull/60841#issuecomment-2065709938) makes me think it is now and these docs just need updating to the latest:

Reply:
Updated by cc75116

I've allowed the `justify` value and updated the documentation as well.